phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2461,11 +2464,11 @@ public Object run() throws Exception {
                         try (PhoenixConnection connection =
                                      QueryUtil.getConnectionOnServer(env.getConfiguration())
                                              .unwrap(PhoenixConnection.class)) {
-                            try{
+                            try {
                                 MetaDataUtil.deleteFromStatsTable(connection, deletedTable,
                                         physicalTableNames, sharedTableStates);
-                                LOGGER.info("Table stats deleted successfully. "+
-                                        deletedTable.getPhysicalName().getString());
+                                LOGGER.info("Table stats deleted successfully. "
+                                    + deletedTable.getPhysicalName().getString());
```

Comment:
same here

Reply:
phoenix code have more this ways that not placeholder in log. I will solved this in another issues.